code/modules/buildmode/bm_mode.dm

```diff
@@ -5,7 +5,7 @@
 
 /datum/buildmode_mode
 	var/key = "oops"
-
+	var/button_icon = 'icons/misc/buildmode.dmi' // SKYRAT ADDITION - remove buildmode duplicate icons, divine tg and skyrat
```

Comment:
This can be defined in the modular file for buildmode mode datums, with the offercontrol one. It should also be using /// to explain what the variable is there for.